### ISSUE TITLE
other pages: ensure proper heading order

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/untd-site#107]
page should have the following heading structure:

- h1: UNTX Social Mobility Hub (site name)
  - h2: The Research Library
  - h2: Research Library Topics
    - h3: Socioeconomic Mobility
    - h3: Family
    - h3: Quality Education
    - h3: Neighborhoods
    - h3: Health and Trauma
    ...

**Comment by Lane** [comment]
made one ticked for all pages, no longer needed.
